etc/logging.yaml.syslog.sample

```diff
@@ -1,38 +0,0 @@
-version: 1
```

Comment:
This file is required, I had asked you to check https://github.com/Tendrl/node-agent/blob/develop/tendrl-node-agent.spec#L49